website/docs/docs/build/cumulative-metrics.md

```diff
@@ -36,6 +36,13 @@ metrics:
 
 ```
 
+## Limitations
+Cumulative metrics are currently under active development and have the following limitations:
+
+1. Cumulative metrics can only be queried using [metric_time](../website/docs/docs/build/dimensions.md) dimension. If you don't use metric_time in the query the cumulative metric will return incorrect results because it won't do the time spine join. This means you cannot refrence time dimensions other then the metric_time in the query.
```

Comment:
```suggestion
1. You can only use the [`metric_time` dimension](/docs/docs/build/dimensions#time) to check cumulative metrics. If you don't use `metric_time` in the query, the cumulative metric will return incorrect results because it won't perform the time spine join. This means you cannot reference time dimensions other than the `metric_time` in the query.
```